baseapp/abci.go

```diff
@@ -175,8 +175,7 @@ func (app *BaseApp) BeginBlock(req ocabci.RequestBeginBlock) (res abci.ResponseB
 	if app.checkState != nil {
```

Comment:
The comments below seem they need an update. I'm not sure. Can you check?

`// we also set block gas meter to checkState in case the application needs to`
`// verify gas consumption during (Re)CheckTx`

Reply:
The comment states about gas meter, not max_gas, so it would be OK to leave it as is.

Reply:
Ok~